selftests/functional/test_loader.py

```diff
@@ -233,11 +230,11 @@ def test_runner_simple_python_like_multiple_files(self):
         cmd_line = "%s list -V %s" % (AVOCADO, mytest)
         result = process.run(cmd_line)
         self.assertIn('SIMPLE: 1', result.stdout)
-        # job should be able to finish under 5 seconds. If this fails, it's
-        # possible that we hit the "simple test fork bomb" bug
+        # Job should finish immediately unless there is a bug in Avocado
+        # which produces a fork-bomb.
```

Comment:
OK, so first, I think you agree that "we have failures that **do not** come from fork-bombs", right?

Then you have just said that "immediately could mean anything... can take an hour...".  So, if we were to follow that logic, should we increase the timeout to > 1 hour? 

Increasing the timeout is a palliative.  5s already looked like something that would never be reached without the fork-bomb bug, but we can not be sure of that on environments we don't control.